pywinauto/application.py

```diff
@@ -587,7 +587,7 @@ def print_control_identifiers(self, depth=None, filename=None):
         log_file = None
         if filename is not None:
             log_file = open(filename, "w")
-            log_func = log_file.write
+            log_func = lambda arg: log_file.write(str(arg) + os.linesep)

```

Comment:
Should be
```python
def log_func(msg):
    log_file.write(str(msg) + os.linesep)
```
According to this warning: https://www.quantifiedcode.com/app/project/gh:pywinauto:pywinauto/diff/fe70a29febedfc009f9c9ad88077fac15a3372ba/a7813da1ff743032c52dfd38ee06029a641cf555/issues